tests/sample.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * SLA bridge and channel creation tests.
+ *
+ */
+"use strict";
+
+/*global describe:false*/
+/*global afterEach:false*/
+
+var util = require('util');
+var Emitter = require('events').EventEmitter;
+
+var mockClient;
+var mockChannel;
+var mockBridge;
+// The channels in existance
+var channels = [];
+// The bridges in existance (should only be one)
+var bridges = [];
+// The channels in the bridges
+var bridge_channels = [];
+// Tests if a bridge has been created or not on a particular test. Resets
+var bridge_created = 0;
+// Mock value for whether the endpoint is valid or not
+var valid = 1;
+// Conditional for whether or not StasisStart has been passed yet
+var pastStasis = 0;
+
+// The mocked up version of the callback error function
+var end= function(err){console.log(err);}
```

Comment:
should be a space after end and after the closing ) of the function signature. I would also move console.log on the next line.